pyproject.toml

```
@@ -1,3 +1,25 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "kyber-py"
+version = "0.1.0"
```

shouldn't this be "1.0.0" ?

we have https://github.com/GiacomoPope/kyber-py/releases/tag/v1.0 ...

---

Yeah I agree

---

Perhaps, something newer since the repo advanced since?
```suggestion
version = "1.0.1.dev0"
```

---

what's the meaning of the dev0 part? I would be ok with 1.0.1

---

it's so that if somebody installs the package from a git checkout, and then updates packages with `pip`, they get the _actual_ released 1.0.1 version

---

Yeah, `1.0.1.dev0 < 1.0.1` during dependency resolution. It's one of the possible pre-releases. See PEP 440 for more details.

---

Some projects advance Git to `.dev0` right after cutting a release and then bump to a stable version before making the next one.